index.js

```diff
@@ -1,4 +1,6 @@
+import parseObjectLiteral from './src/preparse';
 
+export { parseObjectLiteral };
 
 import Provider from './src/provider.js';
-export { Provider }; 
+export { Provider };
```

Comment:
Just cosmetic but I like `export { Provider, parseObjectLiteral } ` I.e. Lumping group exports together -- but I'm open -- what do you think?